runtime/ms-rest-azure/lib/credentials/msiTokenCredentials.js

```diff
@@ -12,25 +12,25 @@ class MSITokenCredentials {
       throw new Error('domain must be a non empty string.');

```

Comment:
Isn't tenantID also called as Domain? I could have used the term `tenantID` but all the other login methods call it `domain`, so I stuck to it for consistency. Unless I'm completely off here.